hack/bootstrap-cluster.sh

```diff
@@ -52,19 +56,20 @@ main() {
         fi
         ;;
     "preview")
-        $ROOT/hack/preview.sh $toolchain $keycloak
+        $ROOT/hack/preview.sh $toolchain $keycloak $obo
         ;;
     esac
 }
 
 print_help() {
-    echo "Usae: $0 MODE [-t|--toolchain] [-kc|--keycloak] [-h|--help]"
+    echo "Usae: $0 MODE [-t|--toolchain] [-kc|--keycloak] [-o|--obo] [-h|--help]"
     echo "  MODE             upstream/preview (default: upstream)"
     echo "  -t, --toolchain  (only in preview mode) Install toolchain operators"
     echo "  -kc, --keycloak  (only in preview mode) Configure the toolchain operator to use keycloak deployed on the cluster"
+    echo "  -o, --obo        (only in preview mode) Install and configure the Observability operator"
```

Comment:
We're also deploying Prometheus federation with that option enabled. I'm not sure if we can come up with a better name for the flag, but the description should also mention something about the Prometheus instance that will be deployed.

Reply:
I changed the description to:  
`(only in preview mode) Install Observability operator and Prometheus instance for federation`